src/pages/experiments/[experimentId]/data-processing/data-integration/components/DataIntegration.jsx

```diff
@@ -84,9 +87,17 @@ const DataIntegration = () => {
   const router = useRouter();
   const { experimentId } = router.query;
 
+  const calculationConfig = useSelector((state) => state.experimentSettings.processing.dataIntegration);
+
   const [activePlotKey, setActivePlotKey] = useState('elbowPlot');
   const [config, setCurrentConfig] = useState(persistedConfigs.elbowPlot);
```

Comment:
These ones are basically having a similar function in terms of state updates and and because of that I think that we can remove the `[config, setCurrentConfig]` state variable. If we keep `[activePlotKey, setActivePlotKey]` (which I would rename so that the name for it matches the name for the state variable that we use for that in the configure embedding for clearness and consistency), then `config` will just be equal to `persistedConfigs[activePlotKey]`.
This will simplify the code, because now you only need to worry about updating `activePlotKey`, the `config` will then be automatically taken care of.

Reply:
@cosa65 can you handle this in your changes? if I fix this in this commit, I'll have to update the plot stylings.

Reply:
Actually removing this useState is part of what I am doing in my current ticket, it was meant to be a temporary thing to allow to test basic styling functionality until we started working on redux.